examples/multiple_tables/main.tf

```
@@ -22,15 +22,29 @@ provider "google" {
 }
 
 module "bigquery" {
-  source            = "../.."
-  dataset_id        = "foo"
-  dataset_name      = "foo"
-  description       = "some description"
-  expiration        = var.expiration
-  project_id        = var.project_id
-  location          = "US"
-  tables            = var.tables
-  time_partitioning = var.time_partitioning
-  dataset_labels    = var.dataset_labels
+  source                     = "../.."
+  dataset_id                 = "foo"
+  dataset_name               = "foo"
+  description                = "some description"
+  expiration                 = var.expiration
+  project_id                 = var.project_id
+  location                   = "US"
+  tables                     = var.tables
+  time_partitioning          = var.time_partitioning
+  dataset_labels             = var.dataset_labels
+  delete_contents_on_destroy = true
+  authorized_views = [
+    {
+      table_full_name = "project.foo.foo"
```

We shouldn't require the full_name for `table` and `view. Instead, let's update the authorized_view module to accept two top-level inputs:
- project
- dataset

Then automatically add them on each passed view.

```suggestion
      table_full_name = "foo"
```